images/linux/scripts/base/apt-get-retry.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Looks like duplicate the apt-retry.sh

Reply:
the first one for apt;
the second for apt-get;

Reply:
Why don't we use symlinks?